guides/graylog2.md

```diff
@@ -95,21 +95,24 @@ We also need to gem-install GELF's Ruby client.
     $ sudo /usr/sbin/td-agent-gem install gelf
 
 Configure `/etc/td-agent/td-agent.conf` as follows.
-
+```
     <source>
       type syslog
+      bind 0.0.0.0
```

Comment:
I think we don't need to write a default value because this is an example and shorter is better.